style.css

```diff
@@ -9,4 +13,152 @@
   --gray-light: hsl(234, 11%, 52%);
   --gray-dark: hsl(233, 14%, 35%);
   --gray-darker: hsl(237, 14%, 26%);
+  --bright-blue: hsl(220, 98%, 61%);
+}
+
+
+main {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+  font-family: "Josefin Sans", sans-serif;
+  color: var(--text-color);
+}
+
+header {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 375px;
+  height: 12.5rem;
+  padding: 1.8rem;
```

Comment:
setting a fixed width and height on an element is never a good idea. It prevents us from creating a responsive design and can cause the content to clip or overflow.

Reply:
I think I can work around not putting  aspecific height, but im not sure about the width.